index.js

```diff
@@ -18,6 +18,7 @@ var Retry = function (options) {
     retryMin: 0,
     retryBase: 1.2,
     retryExponent: 33,
+    retryLimit: -1,
```

Comment:
Would prefer if the default was either `false` or `undefined` – more explicit about the intention of the value